pkgs/development/libraries/libhdhomerun/default.nix

```diff
@@ -1,12 +1,12 @@
-{ stdenv, fetchgit }:
+{ stdenv, fetchurl }:
 
-stdenv.mkDerivation {
-  name = "libhdhomerun-1efbcb";
+stdenv.mkDerivation rec {
+  pname = "libhdhomerun";
+  version = "20200521";
 
-  src = fetchgit {
-    url = "git://github.com/Silicondust/libhdhomerun.git";
-    rev = "1efbcb2b87b17a82f2b3d873d1c9cc1c6a3a9b77";
-    sha256 = "11iyrfs98xb50n9iqnwfphmmnn5w3mq2l9cjjpf8qp29cvs33cgy";
+  src = fetchurl {
+    url = "https://download.silicondust.com/hdhomerun/libhdhomerun_${version}.tgz";
```

Comment:
```suggestion
    url = "https://download.silicondust.com/hdhomerun/libhdhomerun_${stdenv.lib.replaceStrings [ "-" ] [ ""] version}.tgz";
```

Reply:
As stated before I prefer to stick with the official version.